### PR TITLE
Fix/update spec from tests

### PR DESCRIPTION
**Pull request by jellejurre**
Endpoints/changes I've not included here:
https://github.com/vrchatapi/specification-test/blob/main/data/requests/economy/getsteamtransactions.md because it needs oneOf (empty string vs object)

https://github.com/vrchatapi/specification-test/blob/main/data/requests/permissions/getassignedpermissions.md i did all but the data ones since that is arbitrary I think

https://github.com/vrchatapi/specification-test/blob/main/data/requests/system/getjavascript.md because JS
https://github.com/vrchatapi/specification-test/blob/main/data/requests/system/getcss.md because CSS
https://github.com/vrchatapi/specification-test/blob/main/data/requests/system/getinfopush-with-user-all.md because lol
https://github.com/vrchatapi/specification-test/blob/main/data/requests/system/getconfig.md because it's highly unstable and half of these are already outdated

notes:
https://github.com/vrchatapi/specification-test/blob/main/data/requests/groups/getgroup.md i've set as a list of string since it returns [] and I have no idea what that could be

### CODE REVIEW

**Comment by C0D3-M4513R** [comment]
You could add Constants from getConfig I think.
That should not be a random one.

File: openapi/components/schemas/UserExists.yaml [hunk 1]
```diff
@@ -4,12 +4,7 @@ properties:
     default: false
     description: Status if a user exist with that username or userId.
     type: boolean
-  nameOk:
-    default: false
-    description: Is the username valid?
-    type: boolean
 required:
   - userExists
-  - nameOk
```
Comment (ariesclark) [inline openapi/components/schemas/UserExists.yaml:13]: This is still included if you search by username.

File: openapi/components/paths/instances.yaml [hunk 2]
```diff
@@ -100,25 +100,6 @@ paths:
...
-    post:
-      summary: Send Self Invite
-      tags:
-        - instances
-      operationId: sendSelfInvite
-      security:
-        - authCookie: []
-      description: Sends an invite to the instance to yourself.
-      x-codeSamples:
-        $ref: "../codeSamples/instances.yaml#/~1instance~1{worldId}:{instanceId}~1invite/post"
```
Comment (ariesclark) [inline openapi/components/paths/instances.yaml:116]: Can you also delete the related files?

**Review by ariesclark — CHANGES_REQUESTED** [review]

File: openapi/components/paths/instances.yaml (hunk 2 above)
Comment (jellejurre) [reply openapi/components/paths/instances.yaml:116]: I deleted the example and the path

File: openapi/components/schemas/UserExists.yaml (hunk 1 above)
Comment (jellejurre) [reply openapi/components/schemas/UserExists.yaml:13]: Re-added, but removed required flag

File: openapi/components/paths/instances.yaml (hunk 2 above)
Comment (ariesclark) [reply openapi/components/paths/instances.yaml:116]: This one as well.
``$ref: ../responses/instances/InstanceSelfInviteSuccess.yaml``

Comment (jellejurre) [reply openapi/components/paths/instances.yaml:116]: https://github.com/vrchatapi/specification/pull/391/commits/b8c05878ff59de865c20ca77dd69f60a1aade3e3
Didnt i delete that one here?

Comment (ariesclark) [reply openapi/components/paths/instances.yaml:116]: Ah, was reviewing the PR on the GitHub mobile app, must've missed that. Did you run your changes against the test repository?

Comment (jellejurre) [reply openapi/components/paths/instances.yaml:116]: No, good shout, will do

Comment (jellejurre) [reply openapi/components/paths/instances.yaml:116]: Done, should have fixed the issues that exposed

**Review by ariesclark — APPROVED** [review]

**Pull request merged**